config.tfvars

```diff
@@ -130,6 +130,9 @@ bitbucket_helm_chart_version = "1.2.0"
 #bitbucket_admin_display_name  = "<DISPLAY NAME>"
 #bitbucket_admin_email_address = "<EMAIL ADDRESS>"
 
+# The display name of Bitbucket instance
+#bitbucket_display_name = "<DISPLAY NAME>"
```

Comment:
Need a space between `#` and comment

Reply:
If user uses `command`+`/` to uncomment this line, an extra space will make it unaligned... 
And to be consistent with other commented-out variables, see line 130, 131.